README.md

```diff
@@ -189,7 +189,7 @@ jobs:
     name: Erlang/OTP ${{matrix.otp}} / rebar3 ${{matrix.rebar3}}
     strategy:
       matrix:
-        otp: ['20.3', '21.3', '22.2']
+        otp: ['21.3', '21.3', '22.2']
```

Comment:
We could probably remove this. It's a repeat.

Reply:
derpy derp.